bakery/src/tests/data/collection.assembled-metadata.json

```diff
@@ -5,7 +5,7 @@
     "m42092@1.10": {
         "abstract": "Explain the difference between a model and a theory"
     },
-    "col11406@1.17": {
```

Comment:
@rnathuji I want to double check the change in this test data as well as in `data/collection.baked-metadata.json` are correct. iirc, we should never see `colxxxxx@v.v` anymore, correct? (because of the change you made to always include metadata in `neb get`)?

Reply:
Yeah, that's correct.